migration_guides/70.x.x-to-71.0.0-migration-guide.md

```diff
@@ -0,0 +1,389 @@
+                        },
+                        "navigation": {
+                            "subNavigation": {
+                                "overviewURL": "https..."
+                            }
+                        }
+                    })
+                }}
+                NEW
+                {{
+                    onsHeader({
+                        "title": 'Page title',
+                        "mastheadLogoUrl": {
+                            "multipleLogos": {
+                                "logo1": {
+                                    "logoUrl" : "https..."
+                                }
+                            }
+                        },
+                        "titleUrl": '#0',
+                        "language": {
+                            "languages": [
+                                {
+                                    "url": '#0',
+                                    "isoCode": 'en',
+                                    "text": 'English',
+                                    "current": true
+                                },
+                                {
+                                    "url": '#0',
+                                    "isoCode": 'cy',
+                                    "text": 'Cymraeg',
+                                    "current": false
+                                }
+                            ]
+                        },
+                        "navigation": {
+                            "subNavigation": {
+                                "overviewUrl": "https..."
+                            }
+                        }
+                    })
+                }}
+                ```
+
+        </details>
+
+## Components
+
+This section will list all the changes that are specific to one component.
+
+### **_onsCallToAction_**
+
+[**Link to PR**](https://github.com/ONSdigital/design-system/pull/3143)
+
+-   **Description of change:** The `onsCallToAction` component has been removed.
+-   **Reason for change:** The component was removed as it was only developed for use on the Census website.
+-   **Steps:**
+
+    1. Locate all instances of `onsCallToAction` in your codebase.
+    2. You could update any instance of `onsCallToAction` with `onsButton`. The `onsButton` component is not a replacement for the `onsCallToAction` component so transition will not be like for like.
+
+    -   <details>
+          <summary><b>Click for example:</b></summary>
+
+        ```njk
+        OLD
+        {% from "components/call-to-action/_macro.njk" import onsCallToAction %}
+
+            {{-
+                onsCallToAction({
+                    "headingText": 'Call to action heading.',
+                    "paragraphText": 'Descriptive text about call to action',
+                    "button": {
+                        "text": 'Start',
+                        "url": '#0'
+                    }
+                })
+            }}
+        NEW
```

Comment:
Instead of NEW can we call it something like alternate implementation as we are removing the component and providing another one. Sorry, I am not able to remember if we discussed this already in our tech sessions.